tests/sentry/tasks/test_sentry_apps.py

```diff
@@ -313,6 +313,59 @@ def test_integration_org_error_created_doesnt_send_webhook(self, safe_urlopen):
         assert not safe_urlopen.called
 
 
+class TestSendResourceChangeWebhook(TestCase):
+    def setUp(self):
+        self.project = self.create_project()
+        self.sentry_app_1 = self.create_sentry_app(
+            organization=self.project.organization,
+            events=["issue.created"],
+            webhook_url="https://google.com",
+        )
+        self.install_1 = self.create_sentry_app_installation(
+            organization=self.project.organization, slug=self.sentry_app_1.slug
+        )
+        self.sentry_app_2 = self.create_sentry_app(
+            organization=self.project.organization,
+            events=["issue.created"],
+            webhook_url="https://apple.com",
+        )
+        self.install_2 = self.create_sentry_app_installation(
+            organization=self.project.organization,
+            slug=self.sentry_app_2.slug,
+            status=SentryAppInstallationStatus.INSTALLED,
+        )
+
+    @patch("sentry.tasks.sentry_apps.safe_urlopen", return_value=MockResponse404)
+    @with_feature("organizations:integrations-event-hooks")
+    def test_sends_webhooks_to_all_installs(self, safe_urlopen):
+        one_min_ago = iso_format(before_now(minutes=1))
+        event = self.store_event(
+            data={
+                "message": "Foo bar",
+                "exception": {"type": "Foo", "value": "shits on fiah yo"},
+                "level": "error",
+                "timestamp": one_min_ago,
+            },
+            project_id=self.project.id,
+            assert_no_errors=False,
+        )
+
+        with self.tasks():
+            post_process_group(
+                is_new=True,
+                is_regression=False,
+                is_new_group_environment=False,
+                cache_key=write_event_to_cache(event),
+                group_id=event.group_id,
+            )
+
+        assert len(safe_urlopen.mock_calls) == 2
+        call_1 = safe_urlopen.mock_calls[0].kwargs
+        call_2 = safe_urlopen.mock_calls[1].kwargs
+        assert call_1["url"] == self.sentry_app_1.webhook_url
+        assert call_2["url"] == self.sentry_app_2.webhook_url
```

Comment:
if we really wanted we could do something like 

```python
call_urls = [call.kwargs["url"] for call in safe_urlopen.mock_calls]
assert self.sentry_app_1.webhook_url in call_urls
assert self.sentry_app_2.webhook_url in call_urls
```

but im not actually sure if this test has the potential to be flakey or not

Reply:
Your suggestion looks less likely to flake than the existing. Call obj should have url or else it would fail earlier in the test.